src/Microsoft.Security.Utilities.Core/PreciselyClassifiedSecurityKeys/SEC101_200_CommonAnnotatedSecurityKey.cs

```diff
@@ -22,26 +22,41 @@ public override IEnumerable<string> GenerateTruePositiveExamples()
             int count = 0;
             int attempts = 0;
 
-            while(true)
+            foreach (bool longForm in new[] { true, false })
             {
-                foreach (bool longForm in new[] { true, false })
+                while (true) 
                 {
                     char testChar = (char)('a' + attempts++);
-                    string example = IdentifiableSecrets.GenerateCommonAnnotatedTestKey(IdentifiableSecrets.VersionTwoChecksumSeed,
-                                                                                        "TEST",
-                                                                                        customerManagedKey: true,
-                                                                                        platformReserved: null,
-                                                                                        providerReserved: null,
-                                                                                        longForm: false,
-                                                                                        testChar);
 
-                    if (example == null) { continue; }
+                    if (testChar == '{')
+                    {
+                        int z = 10;
+                    }
+
+                    string example;
 
-                    if (++count == 20)
+                    try
+                    { 
+                        example = IdentifiableSecrets.GenerateCommonAnnotatedTestKey(IdentifiableSecrets.VersionTwoChecksumSeed,
+                                                                                     "TEST",
+                                                                                     customerManagedKey: true,
+                                                                                     platformReserved: null,
+                                                                                     providerReserved: null,
+                                                                                     longForm,
+                                                                                     testChar);
+                    }
+                    catch (InvalidOperationException)
                     {
-                        break;
+                        example = null;
                     }
 
+                    if (++count > 26)
```

Comment:
>26 [](http://example.com/codeflow?start=34&length=2)

this kind of magic number is really no good in code. see my other suggestion, just break once we've examined all the chars that are legal for this scenario, lowercase alpha

Reply:
Fixed.